File: core-java-practice/gcr-codebase/PerimeterOfRectangle.java
import java.util.Scanner;
public class PerimeterOfRectangle {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // Input length and breadth
        System.out.print("Enter length: ");
        double length = sc.nextDouble();
        System.out.print("Enter breadth: ");
        double breadth = sc.nextDouble();
        // Perimeter formula
        double perimeter = 2 * (length + breadth);
        // Output result
        System.out.println("Perimeter of Rectangle = " + perimeter);
    }
}

File: core-csharp-practice/gcr-codebase/programming-elements/PerimeterOfRectangle.cs
class PerimeterOfRectangle
{
    static void Main(string[] args)
    {
        // Input length and breadth
        Console.Write("Enter length: ");
        double length = Convert.ToDouble(Console.ReadLine());

        Console.Write("Enter breadth: ");
        double breadth = Convert.ToDouble(Console.ReadLine());

        // Perimeter formula
        double perimeter = 2 * (length + breadth);

        // Output result
        Console.WriteLine("Perimeter of Rectangle = " + perimeter);
    }
}
